Libraries/pbxbuild/Sources/Phase/ProductTypeResolver.cpp

```diff
@@ -268,6 +269,135 @@ ResolveFrameworkStructure(Phase::Environment const &phaseEnvironment, Phase::Con
     return true;
 }
 
+static ext::optional<std::string>
+FindUmbrellaHeaderName(pbxsetting::Environment const &environment, std::string const &moduleName, pbxproj::PBX::Target::shared_ptr const &target)
+{
+    std::string lowerModuleName;
+    std::transform(moduleName.begin(), moduleName.end(), std::back_inserter(lowerModuleName), ::tolower);
```

Comment:
HFS+ is case-insensitive by default, so people get this wrong all the time. For example Pop for a long time had the target name `pop` but the header was called `POP.h`, since it happened to work at the time.